orchestrator/gorc/src/config.rs

```diff
@@ -268,6 +269,7 @@ impl Default for CosmosSection {
             grpc: "http://localhost:9090".to_owned(),
             prefix: "cosmos".to_owned(),
             gas_price: GasPrice::default(),
+            gas_limit: 300000,
```

Comment:
since gas estimate should return something close to `169121` (which is the actual gas used) , would it be better to set the default "minimum" gas_limit to closer/lower than this value like 200000